libs/model-ad/disease-correlation-comparison-tool/src/lib/disease-correlation-comparison-tool.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { BaseComparisonToolComponent } from '@sagebionetworks/explorers/comparison-tools';
+import { ComparisonToolWrapperComponent } from '@sagebionetworks/model-ad/comparison-tool-wrapper';
+
+@Component({
+  selector: 'model-ad-disease-correlation-comparison-tool',
+  imports: [BaseComparisonToolComponent],
+  templateUrl: './disease-correlation-comparison-tool.component.html',
+  styleUrls: ['./disease-correlation-comparison-tool.component.scss'],
+})
+export class DiseaseCorrelationComparisonToolComponent
+  extends ComparisonToolWrapperComponent
+  implements OnInit
+{
+  override isLoading = true;
+  override resultsCount = 40000;
+
+  constructor() {
+    super();
+  }
+
+  ngOnInit() {
+    setTimeout(() => {
+      this.isLoading = false;
+    }, 3000);
+  }
```

Comment:
This boilerplate code is in all 3 CTs (comparison tools) for now.  This placeholder code will be swapped out in the future for the real implementation and was written to showcase proper css styling when loaded.